### PR TITLE
REL: v0.16.0

### PR DESCRIPTION
**Pull request by sappelhoff**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sappelhoff** [comment]
@conda-forge-admin, please rerender

**Comment by sappelhoff** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,26 +19,27 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >={{ python_min }}
```
Comment (hoechenberger) [inline recipe/meta.yaml:22]: Following the docs, this should be 
```suggestion
    - python {{ python_min }}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,26 +19,27 @@ build:
...
     - pandas >=1.3.2
-    - edflib-python >=1.0.6
-    - edfio >=0.2.1
-    - defusedxml
+    - pybv >=0.7.5
+    - pymatreader
 
 test:
+  requires:
+    - python >={{ python_min }}
```
Comment (hoechenberger) [inline recipe/meta.yaml:42]: Same

**Review by hoechenberger — COMMENTED** [review]
@sappelhoff i left two comments

File: recipe/meta.yaml (hunk 1 above)
Comment (sappelhoff) [reply recipe/meta.yaml:22]: Thank you! See

https://github.com/conda-forge/mne-bids-feedstock/commit/dba1b2c0d2383e861184a5ebda1dde6ea57b8955

**Pull request merged**